packages/orchestration/src/utils/start-helper.js

```diff
@@ -26,12 +26,18 @@ import { makeZcfTools } from './zcf-tools.js';
  * @typedef {{
  *   localchain: Remote<LocalChain>;
  *   orchestrationService: Remote<CosmosInterchainService>;
- *   storageNode: Remote<StorageNode>;
+ *   storageNode?: Remote<StorageNode>;
  *   timerService: Remote<TimerService>;
  *   agoricNames: Remote<NameHub>;
  * }} OrchestrationPowers
  */
 
+/**
+ * @typedef {{
+ *   disableStorageNode?: boolean;
```

Comment:
Feedback on the name is encouraged.

Current functionality: when true, `storageNode` is undefined in `provideOrchestration`. As a result:
- chain facades will note create a storage nodes at `nodesPath.${bech32Addr}` for each account
- orch exo account kits will not create a recorderKits and write their address / channel info to the path

This is to say, it's a bit more than "no storage node" - trying to find the words to describe "this won't automatically publish addresses to vstorage or create a topicKit to subscribe to"

Reply:
boolean makes sense but let's flip the default. It's a breaking change but an easy fix.

So then when the writes are expected,
```js
export const start = withOrchestration(contract, { publishAccountInfo: true });
```